tests/E2E/test/campaigns/common_scenarios/category.js

```diff
@@ -113,17 +114,20 @@ module.exports = {
       test('should search for category ', () => client.searchByValue(CategorySubMenu.search_input, CategorySubMenu.search_button, categoryData.name + date_time));
       test('should click on "Edit" action', () => {
         return promise
-          .then(() => client.clickOnAction(CategorySubMenu.update_button))
-          .then(() => client.getParamFromURL('id_category', 2000));
+          .then(() => client.scrollWaitForExistAndClick(CategorySubMenu.update_button))
+          .then(() => client.getURL(2000))
+          .then((res) => global.param['id_category'] = res.value.split('categories/')[1].split("/")[0]);
       });
       test('should check the category name', () => client.checkAttributeValue(CategorySubMenu.name_input, 'value', categoryData.name + date_time));
       test('should check that the image is well displayed', () => client.checkImage(CategorySubMenu.image_link));
       test('should check that the image thumb is well displayed', () => client.checkImage(CategorySubMenu.thumb_link));
       test('should check the category title', () => client.checkAttributeValue(CategorySubMenu.title, 'value', categoryData.meta_title));
       test('should check the category meta description', () => client.checkAttributeValue(CategorySubMenu.meta_description, 'value', categoryData.meta_description));
       test('should check the category friendly url', () => client.checkAttributeValue(CategorySubMenu.simplify_URL_input, 'value', categoryData.friendly_url + date_time));
-      test('should click on "Save" button', () => client.waitForExistAndClick(CategorySubMenu.save_button));
-      test('should click on "Reset" button', () => client.waitForExistAndClick(CategorySubMenu.reset_button));
+      test('should click on "Save" button', () => client.scrollWaitForExistAndClick(CategorySubMenu.save_button));
+      test('should verify the appearance of the green validation', () => client.checkTextValue(CatalogPage.green_validation, 'Successful update.','contain'));
+      test('should click on "Reset" button', () => client.scrollWaitForExistAndClick(CategorySubMenu.reset_button));
+      test('should wait for load',() => client.pause(1500));
```

Comment:
why 1500? why not waiting for a specific element in the dom?

Reply:
We can't use wait for selector to be visible, we should wait for navigation to finish (and we can't do it with selenium).